server/api/apps/shared_models/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```

Comment:
apps.py is not needed if you are directly installing the app in settings.py and not changing other configurations. Alternatively, when installing the app reference the config file.
https://stackoverflow.com/a/46346843